tests/fixtures/modules/invalid-with-statement-in-module.result.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Yup, I'm aware. The problem is that every `ecmaFeatures` test is supposed to throw when not enabled. Since `with` is allowed even when not using a module, it doesn't throw, so the test "fails". So the only real solution is to move the test outside of this directory so it's not bound by this rule.